block/retriever.go

```diff
@@ -50,19 +50,22 @@ func (m *Manager) syncUntilTarget(ctx context.Context, syncTarget uint64) error
 			return err
 		}
 
-		if settlementBatch.StartHeight != currentHeight+1 {
+		//code commented out because it may be the case the start height from the da batch is not exactly the next block from the last received
+		/*if settlementBatch.StartHeight != currentHeight+1 {
 			return fmt.Errorf("settlement batch start height (%d) on index (%d) is not the expected", settlementBatch.StartHeight, currStateIdx)
-		}
+		}*/
 
 		err = m.processNextDABatch(ctx, settlementBatch.MetaData.DA.Height)
 		if err != nil {
 			return err
 		}
 
 		currentHeight = m.store.Height()
-		if currentHeight != settlementBatch.EndHeight {
+
+		//code commented out because it may be the case the current height is higher than da height after applying pending blocks, in case of missing blocks
+		/*if currentHeight != settlementBatch.EndHeight {
```

Comment:
you can simply delete the entire lines. no need for leaving it with comments.

Reply:
done